README.md

```diff
@@ -8,14 +8,19 @@ Most importantly, `cloudwash` provides the CLI utility called '`swach`' that can
 
 cloudwash supports following cloud providers:
 
-* Amazon EC2
-* Google Cloud
-* Microsoft Azure
+| Cloud Providers  | vms    | nics    | discs     | images   | pips    | stacks   |
+|------------------|--------|---------|-----------|----------|---------|----------|
+| Amazon EC2       | &check;| &check; | &check;   |  &check; |  &check;|  &check; |
+| Google Cloud     | &check;| &check; | &check;   |  &cross; |  &cross;|  &cross; |
```

Comment:
@anchalsingh2929 We havent yet added support for Nics and Discs clean ups in GCE it is just provisioned but not there yet!

Also, those things which are not applicable for the the Compute Resource should be having `NA` (means Not applicable) instead of cross. E.g stacks is not applicable for GCE and Azure so just set NA there or any signing string for Not applicabke (like check, cross ).